lib/streamlit/elements/arrow_altair.py

```diff
@@ -48,13 +49,7 @@
 if TYPE_CHECKING:
     from streamlit.delta_generator import DeltaGenerator
 
-# Create and enable streamlit theme
-STREAMLIT_THEME = {"embedOptions": {"theme": "streamlit"}}
-
-# This allows to use alt.themes.enable("streamlit") to activate Streamlit theme.
-alt.themes.register("streamlit", lambda: {"usermeta": STREAMLIT_THEME})
-# We don't want to activate the Streamlit theme for all Altair as default for now.
-# However, the Streamlit theme will be activated as default for our built-in charts.
+# no theme applied to charts
 alt.themes.enable("none")
```

Comment:
nit: I think we can remove this line.

Reply:
Hm when I remove the line, it caused the chart to be 300 px. I kept it so that we can have a larger height.

Reply:
Nit: Maybe it is better to explicitly setting it to 350px for the built in charts.

Reply:
I'm just going to remove this line and let it be 300 and update all the snapshots.

Reply:
The snapshots are not updating so I'm just going to make a new PR... 
![Screen Shot 2022-10-31 at 1 45 07 PM](https://user-images.githubusercontent.com/16749069/199107036-aff5f529-1194-470f-bbf7-c70718f72fea.png)

Reply:
I'm going to merge this but I'll work on this in another PR later this week.